examples/haas.cfg.dev-no-hardware

```diff
@@ -1,4 +1,12 @@
-# This is an example configuration file for HaaS. The ``haas`` command will
+# This is an example configuration file for HaaS developers.  It enables debug
+# output, points client operations to the default port used by "haas serve"
+# (5000), utilizes a sqlite DB in the current directory.  It also removes the
+# need for real nodes by supressing node-oriented non-DB actions like IPMI and
+# headnode operations (by setting dry_run) as well as supressing network
```

Comment:
headnode operations aren't really "node oriented." I would just say something along the lines of "a full hardware environment" or somesuch.